QueryDemo.ipynb

```diff
@@ -0,0 +1,3670 @@
+       " \n",
+       " [186 rows x 12 columns]}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "db.search_string('Cruz18', fuzzy_search=False)  # search for strings exactly matching Cruz19 anywhere in the database\n",
+    "db.search_string('Cruz18', fuzzy_search=False, fmt='pandas')  # as above, but have each table as a pandas dataframe"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9291275e",
+   "metadata": {},
+   "source": [
+    "#### Next exaple is query region, where you can use SkyCoord to create a coordinates object that query_region uses to query the database, in this example the Sources table. Make sure to use as accurate as a coordinate as possible. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "84eca441",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<No columns>\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div><i>Table length=1</i>\n",
+       "<table id=\"table6038848384\" class=\"table-striped table-bordered table-condensed\">\n",
+       "<thead><tr><th>source</th><th>ra</th><th>dec</th><th>epoch</th><th>equinox</th><th>shortname</th><th>reference</th><th>other_references</th><th>comments</th></tr></thead>\n",
+       "<thead><tr><th>str23</th><th>float64</th><th>float64</th><th>object</th><th>object</th><th>str9</th><th>str10</th><th>object</th><th>object</th></tr></thead>\n",
+       "<tr><td>2MASS J09373487+2931409</td><td>144.395292</td><td>29.528028</td><td>None</td><td>None</td><td>0937+2931</td><td>Burg02.421</td><td>None</td><td>None</td></tr>\n",
+       "</table></div>"
+      ],
+      "text/plain": [
+       "<Table length=1>\n",
+       "         source             ra        dec    ... other_references comments\n",
+       "         str23           float64    float64  ...      object       object \n",
+       "----------------------- ---------- --------- ... ---------------- --------\n",
+       "2MASS J09373487+2931409 144.395292 29.528028 ...             None     None"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# An example of query wth an empty table\n",
+    "incorrect_results = db.query_region(SkyCoord(144.3, 29.5, frame='icrs', unit='deg'), output_table='Sources', ra_col='ra', dec_col='dec')  # specifying the name of the table with coordinate information\n",
+    "print(incorrect_results)  \n",
+    "\n",
+    "# An example that queries the database correctly \n",
+    "correct_results = db.query_region(SkyCoord(144.395292, 29.528028, frame='icrs', unit='deg'), output_table='Sources', ra_col='ra', dec_col='dec')  # specifying the name of the table with coordinate information\n",
+    "correct_results"
```

Comment:
You may want to rework your outputs here. The end result is a table but you've printed both the incorrect and (implicitly) the correct results. So which of them is it? Better to print out a string with some text saying something along the lines of Number of incorrect results: len(incorrect_results) or such to be clear.